compiler/cli/src/fuzz.rs

```diff
@@ -37,11 +37,11 @@ pub(crate) fn fuzz(options: Options) -> ProgramResult {
     } else {
         error!("");
         error!("Finished fuzzing.");
-        error!("These are the failing cases:");
-        for case in failing_cases {
-            error!("");
-            case.dump(&db);
-        }
+        // error!("These are the failing cases:");
+        // for case in failing_cases {
+        //     error!("");
+        //     case.dump(&db);
+        // }
```

Comment:
_Why_ is this commented out?

Reply:
Printing the stack trace currently segfaults for some reason. That also happens on main, so this PR is still an improvement, but I commented the segfaulting code out so that fuzzing at least works (it does report the function and failing inputs).
If you want to, I can also uncomment this so we don't forget to fix the segfault.

Reply:
Done: https://github.com/candy-lang/candy/issues/458